test_utils/pre_commit/add_translations.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+from typing import Sequence
+
+try:
+	from frappe.translate import get_untranslated, update_translations
+except Exception as e:
+	raise(e)
+
+
+def add_translations(lang, app):
+	untranslated_file = "untranslated_strings"
```

Comment:
@Alchez Suggests wrapping this entire function in a try/except that allows it to continue in the pre-commit context